### PR TITLE
Add coop warning

### PR DESCRIPTION
**Pull request by cb-jake**
### _Summary_

Add additional coop warning for developers. This is meant to help resolve coop issues

### _How did you test your changes?_

1. Spin up the playground
2. Set the next.config.js COOP header to same-origin
3. Verify the warning is present inside the console when running the app.
4. You'll know the coop is properly set if you see the coop warning when interacting with the popup

https://github.com/user-attachments/assets/c5440e3c-20b5-4d50-be22-cfd111b06d73

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

File: examples/testapp/next.config.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+ */
+module.exports = {
+  async headers() {
+    return [
+      {
+        source: "/",
+        headers: [
+          {
+            key: "Cross-Origin-Opener-Policy",
+            value: "unsafe-none", // Or 'unsafe-none' if you want to disable COOP
```
Comment (spencerstock) [inline examples/testapp/next.config.js:12]: nit: The comment here seems outdated

**Review by spencerstock — DISMISSED** [review]
Looks good

File: examples/testapp/next.config.js (hunk 1 above)
Comment (cb-jake) [reply examples/testapp/next.config.js:12]: Thanks. Let me fix real quick

**Review by spencerstock — APPROVED** [review]
Looks good

**Comment by cb-heimdall** [comment]
Review Error for spencerstock @ 2024-10-01 18:09:58 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

**Review by spencerstock — APPROVED** [review]

**Pull request merged**